x/stake/handler_sidechain.go

```diff
@@ -278,6 +278,9 @@ func handleMsgEditSideChainValidatorWithVoteAddr(ctx sdk.Context, msg MsgEditSid
 	}
 
 	if len(msg.SideVoteAddr) != 0 {
+		if sdk.IsUpgrade(sdk.BCFusionFirstHardFork) {
```

Comment:
why? After the first hardfork, we should still allow for vote, dont we?

Reply:
fixed, will disable MsgEditSideChainValidatorWithVoteAddr after fistr hard fork